Makefile

```diff
@@ -168,21 +168,13 @@ test-db/12-create.sql: pyramid_oereb_standard.yml .venv/install-timestamp
 test-db/13-fill.sql: pyramid_oereb_standard.yml .venv/install-timestamp
 	$(VENV_BIN)python pyramid_oereb/standard/load_sample_data.py --configuration $< --sql-file $@
 
-.PHONY: serve-db-dev
-serve-db-dev: tests-docker-drop-db test-db/12-create.sql test-db/13-fill.sql
-	docker build -t $(DOCKER_BASE)-db-dev test-db
-	docker run --rm --name $(DOCKER_CONTAINER_BASE)-db-dev \
-	    --publish=5432:5432 \
-	    --env=POSTGRES_DB=pyramid_oereb \
-	    $(DOCKER_BASE)-db-dev
-
-.PHONY: serve-print-example
-serve-print-example:
-	docker build -t $(DOCKER_BASE)-print-dev print
-	docker run --rm --name $(DOCKER_CONTAINER_BASE)-print-dev \
-	    --publish=8280:8080 \
-	    $(DOCKER_BASE)-print-dev
-
 .PHONY: deploy
 deploy:
 	$(VENV_BIN)python setup.py sdist bdist_wheel upload
+
+logo_%.png: pyramid_oereb_standard.yml
+	touch --no-create $@
```

Comment:
I don't understand why this is needed

Reply:
The rule who create `pyramid_oereb_standard.yml` also creates the required logos, and If I don't add a command it don't works :-)